src/js/button.js

```diff
@@ -7,13 +12,13 @@
  * @class
  * @constructor
  */
-vjs.Button = vjs.Component.extend({
+var Button = Component.extend({
   /**
    * @constructor
    * @inheritDoc
    */
   init: function(player, options){
```

Comment:
Yep, the plugin should be logging profusely to let users know what changed and what to do instead.
Use the [jquery-migrate](https://github.com/jquery/jquery-migrate/) plugin for inspiration.